portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <title>MM Portfolio</title>
+    <link rel="stylesheet" href="blog_style.css">
+    <script src="blog_script.js"></script>
```

Comment:
I still think you'll need the leading slashes on the css and js resources here in order for the container to render the style correctly. Have you tried invoking `mvn package appengine:run` to verify this?

Reply:
Yes I have, but I can definitely add the ./ to for style purposes!

Reply:
Huh - when I tried this yesterday with Bianca's repo it wasn't rendering when running in the container - but indeed I am incorrect :). Just tried it for my own sanity and it seems to work either way. I would rever that change.